packages/ia-components/sandbox/audio-players/audio-player.stories.js

```diff
@@ -1 +1,2 @@
 import style from './audio-player.less';
+import styles from './youtube-player.less'
```

Comment:
We should rename these two variables to something more easily identifiable, like `audio_styles` and `youtube_styles`